### ISSUE TITLE
Verify OSNotificationPayload structure in Android

### ISSUE THREAD

**Issue by jvandijk** [williamrijksen/com.williamrijksen.onesignal#29]
A title is an optional field for a notification. This means that its existence should be verified before use, else this happens:
```
java.lang.NullPointerException: Attempt to read from field 'java.lang.String com.onesignal.OSNotificationPayload.title' on a null object reference
	at com.williamrijksen.onesignal.ComWilliamrijksenOnesignalModule$NotificationOpenedHandler.notificationOpened(ComWilliamrijksenOnesignalModule.java:96)
	at com.onesignal.OneSignal$10.run(OneSignal.java:1014)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:7325)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by chmiiller** [comment]
Good catch! Is this only happening on Android or on iOS as well?

**Comment by chmiiller** [comment]
Nice, just saw #30 🎉

**Comment by williamrijksen** [comment]
Closed because of the merging of #30.
